Misc/NEWS.d/next/Documentation/2025-10-02-11-51-34.gh-issue-139495.y4Orqf.rst

```diff
@@ -0,0 +1,2 @@
+Fix :func:`hashlib.file_digest` versionchanged description of
```

Comment:
There is no need for a NEWS entry I think. You can remove this file.

Reply:
Yeah, it's unfortunate that `make patchcheck` actually tells you otherwise sometimes. We ought to improve it but nobody has really time for that (and it kinda duplicates that the CI tells you sometimes...).